models/enums_and_bitflags.py

```diff
@@ -418,12 +418,10 @@ class BitFlagConverter( EnumConverter ):
     _dict_inverse = None
 
 
-# the list of possible processing steps from a section of an exposure up to measurments, r/b scores, and report
+# the list of possible processing steps from a section of an exposure up to measurements, r/b scores, and report
 process_steps_dict = {
     1: 'preprocessing',  # creates an Image from a section of the Exposure
     2: 'extraction',     # creates a SourceList from an Image, and a PSF
-    3: 'astro_cal',      # creates a WorldCoordinates from a SourceList
-    4: 'photo_cal',       # creates a ZeroPoint from a WorldCoordinates
```

Comment:
I don't propose actually changing this because it would be painful at this point, but the name 'extraction' really isn't right for this step any more.  Back in the SCP days we would have called this 'reduction', but many astronomers use 'reduction' to mean what we mean by 'preprocessing', so I wouldn't suggest that name.  We're doing more than extraction in this step now, though.  (We were before too, as there was psf estimation.)